docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,56 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
```

Comment:
@iancoffey since Deis < 1.9.0 won't support `deisctl upgrade-prep` and 1.9.0 will if this is merged, the instructions documented here won't actually _work_ for anyone until upgrading from m >= 1.9.0 to n > m.

Would it make sense to _not_ include this documentation until the release after this feature is rolled out?  Otherwise we may have people attempting an upgrade procedure that is guaranteed to fail.